Purchases/Purchasing/Purchases.swift

```diff
@@ -251,8 +251,8 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
                      appUserID: String?,
                      userDefaults: UserDefaults? = nil,
                      observerMode: Bool = false,
-                     platformFlavor: String? = nil,
-                     platformFlavorVersion: String? = nil,
```

Comment:
Oooh these use to be in the configure methods?
The new initialization with static is weird, but I like that it doesn't get exposed in configure for all users so it allows the hybrids to do it. I think it's good.